armory/data/adversarial/ucf101_mars_perturbation_and_patch_adversarial_112x112.py

```diff
@@ -249,7 +255,11 @@ def _generate_examples(self, data_dir_path):
                         "adversarial_perturbation": adv_pert,
                         "adversarial_patch": adv_patch,
                     },
-                    "label": label,
+                    "labels": {
+                        "clean": label,
+                        "adversarial_perturbation": label,  # untargeted, label not used
+                        "adversarial_patch": labels[(i + 1) % len(_LABELS)],  # targeted
```

Comment:
That's a good catch. The target labels were generated according to the order in `_LABELS`.

The following are the differences between `_LABELS` and `labels.sort()`:

`_LABELS` `labels.sort()`
Hammering HammerThrow
HammerThrow Hammering
JumpingJack JumpRope
JumpRope JumpingJack

Since the set of `labels` is the same as `_LABELS`, I think one way to correct the bug is to replace

`labels.sort()` with `labels = _LABELS`. In that case, https://github.com/yusong-tan/armory/blob/90ba4ac31b4e2e8fae6d6b21c704e4aad2087cd6/armory/data/adversarial/ucf101_mars_perturbation_and_patch_adversarial_112x112.py#L206-L208 can be deleted.